SketchUp/sketchup.rb

```diff
@@ -188,12 +184,12 @@ def self.display_name_from_action(action_name)
   #
   # @example
   #   extensions = Sketchup.extensions
-  #   for extension in extensions
+  #   extensions.each{ |extension|
   #     UI.messagebox('The next extension is named: ' + extension.name +
```

Comment:
Would you mind converting this `UI.messagebox` to `puts`? Especially in loops these messageboxes are a pain.